types/type-specs.tsx

```diff
@@ -51,8 +52,14 @@ const invalidParams: Params = { id: 13 }; // $ExpectError
 /*
  * Link and Redirect component type specs
  */
+
+// `to` and `href` are aliases, but they are mutually exclusive and
+// can't be used at the same time:
 <Link to="/users">Users</Link>;
 <Link href="/about">About</Link>;
+<Link href="/about" to="/app" children="" />; // $ExpectError
+<Link children="" />; // $ExpectError
```

Comment:
delete

Reply:
Same here. By specifying `// $ExpectError` we tell linter that this line is incorrect in terms of types: either `to` or `href` should be present.